package.json

```diff
@@ -30,57 +30,55 @@
   "private": true,
   "pnpm": {
     "overrides": {
-      "jsdom@^20.0.3>nwsapi@^2.2.0": "2.2.9",
-      "vite>@types/node": "^20.17.16",
       "electron-vite@^3.0.0>esbuild@0.24.2": "^0.25.0"
     }
   },
   "devDependencies": {
     "@gravitational/build": "workspace:*",
-    "@storybook/addon-actions": "^8.5.3",
-    "@storybook/addon-controls": "^8.5.3",
-    "@storybook/addon-toolbars": "^8.5.3",
-    "@storybook/components": "^8.5.3",
-    "@storybook/preview-api": "^8.5.3",
-    "@storybook/react": "^8.6.2",
-    "@storybook/react-vite": "^8.6.2",
-    "@storybook/test-runner": "^0.21.0",
+    "@storybook/addon-actions": "^8.6.3",
+    "@storybook/addon-controls": "^8.6.3",
+    "@storybook/addon-toolbars": "^8.6.3",
+    "@storybook/components": "^8.6.3",
+    "@storybook/preview-api": "^8.6.3",
+    "@storybook/react": "^8.6.3",
+    "@storybook/react-vite": "^8.6.3",
+    "@storybook/test-runner": "^0.22.0",
     "@testing-library/jest-dom": "^6.6.3",
     "@testing-library/react": "^16.2.0",
     "@testing-library/user-event": "^14.6.1",
     "@types/history": "^4.7.11",
     "@types/jest": "^29.5.14",
-    "@types/node": "^20.17.16",
+    "@types/node": "^20.17.22",
     "@types/react": "^18.3.12",
     "@types/react-dom": "^18.3.1",
     "@types/react-highlight": "^0.12.8",
     "@types/react-router": "^5.1.20",
     "@types/react-router-dom": "^5.1.1",
     "@types/react-transition-group": "^4.4.12",
     "@types/wicg-file-system-access": "^2023.10.5",
-    "cspell": "^8.17.3",
+    "cspell": "^8.17.5",
     "jest": "^29.7.0",
     "jsdom-testing-mocks": "^1.13.1",
-    "msw": "^2.7.0",
+    "msw": "^2.7.3",
     "msw-storybook-addon": "^2.0.4",
     "playwright": "^1.50.1",
-    "prettier": "^3.4.2",
+    "prettier": "^3.5.3",
```

Comment:
@avatus Everything looks okay, but the update to Prettier requires `e/web/teleport/src/Integrations/IntegrationEnroll/PluginEnroll/MultiStep/Entra/RunScript.tsx` to be reformatted. Could you take care of it?

Reply:
yeah i have it ready to go!